autonomic-plugin-common/src/main/java/br/com/autonomiccs/autonomic/plugin/common/services/AutonomiccsSystemVmDeploymentService.java

```diff
@@ -24,6 +24,7 @@
 
 import java.util.ArrayList;
```

Comment:
why are you trying to introduce this class? It is already there at the master branch.
do you have some other change here?